openlibrary/plugins/openlibrary/js/Browser.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Convert url paramaters to an object
+ * @param {String} urlSearch everything after (and including) the '?' of the url
+ * @returns {{ [param: string]: string }}
+ */
+export function getJsonFromUrl(urlSearch) {
+    const query = urlSearch.substr(1);
+    const result = {};
+    if (query) {
+        query.split('&').forEach(part => {
```

Comment:
@jdlrobson : Probably worth function of its own.

Reply:
This is just a copy of the original; I've made some improvements but don't want to spend a ton of time make improvements to the pre-existing code otherwise I'll be here forever :P So opting for won't fix.